packages/features/bookings/Booker/components/OverlayCalendar/OverlayCalendarSwitch.tsx

```diff
@@ -20,7 +20,7 @@ export function OverlayCalendarSwitch({ enabled, hasSession, onStateChange }: Ov
     (state) => state.setCalendarSettingsOverlayModal
   );
   const layout = useBookerStore((state) => state.layout);
-  const switchEnabled = enabled;
+  const switchEnabled = hasSession || enabled;
```

Comment:
This is enabled, but does it overlay the calendar? Please share a Loom with the connected calendar

Reply:
Just a thought – by setting this as const switchEnabled = hasSession || enabled;, wouldn't it always return true for signed-in users, potentially causing the button to never close?

Reply:
> This is enabled, but does it overlay the calendar? Please share a Loom with the connected calendar

did exactly that here:
https://github.com/calcom/cal.com/pull/18191